src/scales/categoryScale.ts

```diff
@@ -142,7 +142,10 @@ export module Scale {
     }
 
     public copy(): Category {
-      return new Category(this._d3Scale.copy());
+      return new Category(this._d3Scale.copy())
+                  .range(this.range())
+                  .innerPadding(this.innerPadding())
+                  .outerPadding(this.outerPadding());
```

Comment:
I think we should separate out this functionality unless necessary.  Unless you want to make a new pull request with this change